README.md

```diff
@@ -53,6 +53,12 @@ The code is written such that as a user you only have to interact with [*main.m*
 
 All user-defined [settings](./Documentation/SettingsOverview.md) are stored in structure *S*. In main.m you have to specify the required settings and are free to change/add the optional settings. 
 
+The [examples folder](./Examples/) contains scripts illustrating how to use settings to:
+- Predict a gait pattern resulting from a limited number of muscle synergies.
+- Predict gait with assistance from an ankle exoskeleton.
+- Run a sensitivity analysis.
+
```

Comment:
Wouldn't this fit better after the section 'Before running a simulation'?

Reply:
Now it's under "how to use the code"